resources/views/vendor/backpack/base/inc/sidebar_content.blade.php

```diff
@@ -39,7 +39,9 @@
 <li><a href="{{ backpack_url('events') }}"><i class="fa fa-calendar"></i> <span>Events</span></a></li>
 @endcan
 
+@can('edit events')
```

Comment:
Shouldn't this be `edit colleagues`?

Reply:
Well, yes.
I slipped by me because the role contains both.

Reply:
Fixed in 59c4ade